src/pages/Applicant/EditableProfile/EditableDetail/container.tsx

```diff
@@ -107,23 +95,21 @@ const EditableDetailContainer: FunctionComponent = () => {
 
   if (applicantError || translationsError) history.push(RoutesBuilder.notFound);
 
+  if (loadingApplicant || loadingTranslations) return <LoadingSpinner/>;
+
+  const [ titleTranslation ] = getTranslations;
+  const translations: IApplicantDetailEditableTranslations = {
```

Comment:
Es necesario definir el tipo?

Reply:
si le pasas un hash generico es mas propenso a errores, con esto sabés que lo que obtenes es un string not undefined en compile time

Reply:
El loading y el error de las translations deberian encargarse de eso igual. Pero bueno, tampoco me molesta mucho

Reply:
![image](https://user-images.githubusercontent.com/20487262/78410029-a5531b80-75e1-11ea-8bf6-9e47aed3ae72.png)

Reply:
Pero typescript es inteligente interpretando tipos y si te fijas, al sacarle el tipo en la variable no tira ningun warning despues de la transpilacion. Si no pudiera hacerlo se quejaria que a esa prop le estas pasando cualquier cosa

Reply:
aaah decis que resolvía el tipo sin meterle la interfaz, ahi pruebo... creo que si getTranslations dice que da un array de strings not null debería resolver sin interfaz, tenes razon

Reply:
ahi lo pruebo